src/utils/filter.ts

```diff
@@ -73,19 +76,19 @@ export const makeCampRoomHitFilter = ({
 };
 
 export const filterUnreadPrivateChats = (
-  chats: WithId<PrivateChatMessage>[],
-  user: FirebaseReducer.AuthState | undefined
+  chats: readonly WithId<PrivateChatMessage>[],
+  user?: FirebaseReducer.AuthState
 ) => {
-  return (
-    chats
-      ?.filter(
-        (message) =>
-          message.from !== user?.uid &&
-          message.deleted !== true &&
-          message.type === "private" &&
-          message.ts_utc.seconds > HIDE_BEFORE &&
-          message.isRead === false
-      )
-      .sort(chatSort) ?? []
-  );
+  if (!chats) return [];
+
+  return chats
+    ?.filter(
```

Comment:
[nit] since `chats` is explicitly checked by the guard above, we shouldn't need the null-safe syntax anymore:

```suggestion
  return chats.filter(
```